2_crosswalk_munge/src/spatial_crosswalks.R

```diff
@@ -188,3 +188,38 @@ buffer_sf_lakes <- function(out_ind, lake_ind, buffer_width){
   gd_put(out_ind, data_file)
 
 }
+
+us_counties_to_states <- function(ind_file, us_counties_ind) {
+  us_counties_sf <- readRDS(sc_retrieve(us_counties_ind))
+  conus_states <- group_by(us_counties_sf, state) %>%
+    summarise() %>%
+    st_transform(crs = 4326)
+
+  # write, post, and promise the file is posted
+  data_file <- scipiper::as_data_file(ind_file)
+  saveRDS(conus_states, data_file)
+  gd_put(ind_file, data_file)
+}
+
+# Intersects lake polygons with state polygons to match each lake to
+# one or more states. If the lake doesn't match any (e.g. it's in
+# Canada), it is given an `NA` in the `state` column.
+crosswalk_lakes_intersect_states <- function(ind_file, lakes_ind, states_ind) {
```

Comment:
I'm just wondering if you want to skip having two targets, and put the `us_counties_to_states` call within this function. I know it would be normal to separate, but the actual meat of `us_counties_to_states` is pretty small and most of the code is the cache handling components, like the sc_retrieve/read, then save/write/put. 

What if instead you had this function accept `us_counties_ind` and had a TRUE/FALSE arg for summarize to county vs state? (I could see a future need for a county overlap target and you could use this function for both).

Reply:
Oh, I like that flexibility! I will work on editing

Reply:
Updated! I also deleted the build yml, indicator, and GD data file related to that old target